servicelb.json

```diff
@@ -0,0 +1,44 @@
+{
+        "name": "contivModel",
+        "objects": [
+                {
+                 "name": "serviceLB",
+                 "type": "object",
+                 "key": ["serviceName","tenantName"],
+                 "properties": {
+                                "serviceName": {
+                                             "type": "string",
+                                             "title": "service name",
+                                             "length": 256
+                                },
+                                "tenantName": {
+                                        "type": "string",
+                                        "title": "Tenant Name",
+                                        "length": 64
+                                },
+
+                                "network": {
+                                             "type": "string",
+                                             "title": "Service subnet",
+                                             "length": 15
```

Comment:
minor: This attribute title should be "Service network name" and length should be 64.